### PR TITLE
docs: fix broken links in `overview.md`

### LINKED ISSUE #2814: doc: r1 overview's link to custom scalar is not useful
https://github.com/tensorflow/tensorboard/blob/master/docs/r1/overview.md#can-i-customize-which-lines-appear-in-a-plot for "custom scalar plugin" links to https://github.com/tensorflow/tensorboard/blob/master/docs/r1/tensorboard/plugins/custom_scalar which is does not exist.

### PR DESCRIPTION
**Pull request by LexusH**
Fixed 2 broken links that were pointed out in #2814.

Fixes #2814.

### CODE REVIEW

File: docs/r1/overview.md [hunk 1]
```diff
@@ -311,15 +311,15 @@ method.
 
 ### Can I customize which lines appear in a plot?
 
-Using the [custom scalars plugin](tensorboard/plugins/custom_scalar), you can
+Using the [custom scalars plugin](https://github.com/tensorflow/tensorboard/tree/master/tensorboard/plugins/custom_scalar), you can
```
Comment (stephanwlee) [inline docs/r1/overview.md]: Could you use a relative path? i.e., `../../../tensorboard/plugins/custom_scalar`. Here and below

Comment (LexusH) [reply docs/r1/overview.md]: Sure! I'll work on fixing the other two broken links you pointed out too!

File: docs/r1/overview.md [hunk 2]
```diff
@@ -311,15 +311,15 @@ method.
 
 ### Can I customize which lines appear in a plot?
 
-Using the [custom scalars plugin](tensorboard/plugins/custom_scalar), you can
+Using the [custom scalars plugin](/../../tree/master/tensorboard/plugins/custom_scalar), you can
```
Comment (stephanwlee) [inline docs/r1/overview.md]: Remove tree/master and the leading "/"

File: docs/r1/overview.md [hunk 3]
```diff
@@ -335,7 +335,7 @@ TensorBoard uses [reservoir
 sampling](https://en.wikipedia.org/wiki/Reservoir_sampling) to downsample your
 data so that it can be loaded into RAM. You can modify the number of elements it
 will keep per tag in
-[tensorboard/backend/application.py](tensorboard/backend/application.py).
+[tensorboard/backend/application.py](/tensorboard/backend/application.py).
```
Comment (stephanwlee) [inline docs/r1/overview.md]: Please remove "/" and use relative path instead.

File: docs/r1/overview.md (hunk 1 above)
Comment (wchargin) [reply docs/r1/overview.md]: Thank you! For context, the reason that we like relative links is that
they enable editors to cross-link. For instance, in Vim, I can just
press `gf` (go-file) with my cursor on the relative link and it’ll open
in my editor.

**Pull request merged**